plugin/dnstap/README.md

```diff
@@ -63,6 +64,14 @@ dnstap /tmp/dnstap.sock {
 }
 ~~~
 
+Log to a socket, customize the "extra" field in dnstap payload.
+
+~~~ txt
+dnstap /tmp/dnstap.sock {
+  extra forward/upstream
```

Comment:
I think, this should be configured as `{/forward/upstream}` if we want to specify metadata label. At least, other plugins (rewrite, log) use this format, and this is what package https://github.com/coredns/coredns/tree/master/plugin/pkg/replacer expects.

The text `forward/upstream` still can be interpreted as a constant string (literal).